Source/PeleC.H

```diff
@@ -50,6 +54,9 @@ enum sources { ext_src = 0,
 #endif
                diff_src,
                les_src,
+#ifdef SOOT_MODEL
+	       soot_src,
```

Comment:
I guess we have to run `clang-format` on this code. Looks like the CI isn't passing.